--- a/icd_dispatch.c	2012-08-06 08:53:38.000000000 +0200
+++ b/icd_dispatch.c	2015-09-07 00:05:41.234375000 +0200
@@ -2040,7 +2040,7 @@
     cl_platform_id                  platform,
     cl_uint                         num_media_adapters,
     cl_dx9_media_adapter_type_khr * media_adapters_type,
-    void *                          media_adapters[],
+    void *                          media_adapters,
     cl_dx9_media_adapter_set_khr    media_adapter_set,
     cl_uint                         num_entries,
     cl_device_id *                  devices,
